docs/.vitepress/theme/custom.css

```
@@ -50,25 +46,22 @@
-        padding-top: calc(
-            var(--vp-nav-height) + var(--announcement-offset, 0px)
-        );
+        padding-top: calc(var(--vp-nav-height) + var(--announcement-offset, 0px));
     }
 
     .aside-container {
         padding-top: calc(
-            var(--vp-nav-height) + var(--vp-layout-top-height, 0px) +
-                var(--vp-doc-top-height, 0px) + 48px +
-                var(--announcement-offset, 0px)
+            var(--vp-nav-height) +
```

Stylelint (17.12.0)

[error] Unexpected newline after "+" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)